### ISSUE TITLE
Plz update etc.

### ISSUE THREAD

**Issue by badgermedic76** [Chocohead/OptiFabric#1003]

**Comment by RealMCreeper** [comment]
I agree :)     I need 1.19.4 but the action builds are for 1.19.3

**Comment by ghost** [comment]
Give the Dev time, modding is harder than you think

**Comment by RainbowTea** [comment]
Ya'll asking for the dev to update in the help section is so rude. Also its rude to tell someone to update their FREE service.

**Comment by YellowGreenCraft** [comment]
You are absolutely right, it would be nice if we could get an update on an ETA though, no pressure 😊

**Comment by ghost** [comment]
There is now a beta on curseforge
